aws-powertools/powertools-lambda-typescript
# feat(tracer): middy middleware

## Description of your changes

* [x] Implement middy-based middleware with feature parity against captureLambdaHandler decorator
* [ ] <del>Implement middy-based middleware with feature parity against captureMethod decorator</del>
* [x] Documentation & JSDocs updates
* [x] Implementation unit tests

### How to verify this change

Checkout branch & run `npm run tests` in `src/packages/tracing` and/or observe actions results.

### Related issues, RFCs


[#295](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/295)
[#282](https://github.com/awslabs/aws-lambda-powertools-typescript/discussions/282)

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
